utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -142,6 +143,10 @@ else if (networkSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
       gridPane.add(attachmentLabel, 1, 0);
       gridPane.add(connectivityImageView, 0, 1);
       gridPane.add(connectivityLabel, 1, 1);
+      gridPane.setMaxWidth(80);
+      gridPane.setMaxHeight(100);
+      gridPane.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
```

Comment:
these should match the original settings from the offending css: width and height need switched around and for the background to the white transparent not black. 

```suggestion
      gridPane.setMaxWidth(100);
      gridPane.setMaxHeight(80);
      gridPane.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255,255,255,0.3)"), CornerRadii.EMPTY,
```